text/trace/0235-sampling-threshold-in-trace-state.md

```diff
@@ -0,0 +1,99 @@
+# Sampling Threshold Propagation in TraceState
+
+## Motivation
+
+Sampling can theoretically take place at nearly any point in a distributed tracing system. If sampling is to be performed at multiple points in the process, the only way to reason about it effectively is to make sure that the sampling decisions are **consistent**.
```

Comment:
In this statement "If sampling is to be performed at multiple points in the process", I feel there's a bit of ambiguity on if it is talking about sampling of the same span or sampling of different spans in a trace. It may be good to take the "Consistent probability sampling" motivation/definitions from the earlier experimental OTEP.

Reply:
I would very much appreciate your suggestion on exactly what parts to use. I have read it a couple of times and I'm not finding any parts that seemed like they would fit the description. 

Will you please do one of these?
* Give me a list of specific points you think should be made, and I'll try to write them out
* Copy or write new text and I'll incorporate it